src/pkg/bundler/fetcher/local.go

```diff
@@ -69,10 +69,7 @@ func (f *localFetcher) GetPkgMetadata() (zarfTypes.ZarfPackage, error) {
 		return zarfTypes.ZarfPackage{}, err
 	}
 	defer func(path string) {
-		err := os.RemoveAll(path)
-		if err != nil {
-
-		}
+		os.RemoveAll(path) //nolint:errcheck
```

Comment:
The purpose of wrapping `os.RemoveAll` in `defer` was to do the error check as identified by the linter. But I agree it's unnecessary! What do you think of removing the outer wrapping entirely and just doing:
```go
defer os.RemoveAll(path) //nolint:errcheck
```

Reply:
I agree. Took care of it. Thanks